.github/workflows/msktutil-freebsd.yml

```diff
@@ -28,15 +28,13 @@ jobs:
         run: |
           freebsd-version
           # basic dependencies
-          pkg install -y autoconf cyrus-sasl gmake openldap-sasl-client
+          pkg install -y autoconf cyrus-sasl2-gssapi gmake openldap-sasl-client
           case "${{ matrix.cfg.cc-version }}" in
             gcc)
-              pkg install -y gcc
```

Comment:
This now looks wrong. FreeeBSD does not come with GCC. Did you try that? Has the vbox image GCC preinstalled?

Reply:
you told me to delete both :-D . I am testing it on my fork.

Reply:
I told to delete llvm only.

Reply:
Also rather pull an explicit version because no port will install `gcc` executable, but a suffixed one. E.g., `gcc9`, `gcc10`, etc.

Reply:
Hi @michael-o  , I would suggest that we allow this change to go in. 
All the distributions are compiling with their default gcc or clang compiler versions, determined by their respective policies. IMO this is a fairly sane approach, because this is was users are doing.

We aren't doing a port here. 

I see no point in trying every variant.